background.ts

```diff
@@ -35,7 +35,7 @@ Electron.app.setPath('cache', paths.cache);
 Electron.app.setAppLogsPath(paths.logs);
 
 const console = Logging.background;
-
+const commandLineArgs = Electron.app.isPackaged ? process.argv.slice(1) : process.argv.slice(5);
```

Comment:
A comment on where `1` and `5` came from would be nice.

```suggestion
const commandLineArgs = process.argv.slice(Electron.app.isPackaged ? 1 : 5);
```

Reply:
I've moved this into a separate module, mainly to keep the details out of `background.ts`. Added comments justify the decisions for each of package runs, dev runs, and e2e tests.